spec/support/tasks.cr

```diff
@@ -24,3 +24,35 @@ class AnotherTask < LuckyCli::Task
   def call
   end
 end
+
+class TaskWithParam < LuckyCli::Task
+  summary "This task has a param"
+  param model_name
```

Comment:
No type added because a param will always be a string. If the user needs a number then they can parse and typecast as needed.